portfolio.html

```diff
@@ -0,0 +1,442 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="uper-bar">
+          <div>
+            <h2 class="time">9.41</h2>
+          </div>
+          <div class="symbols">
+            <ul>
+              <li>
+                <img
+                  class="images"
+                  src="bar/Cellular.png"
+                  alt="cellular network symbole"
+                />
+              </li>
+              <li>
+                <img class="images" src="bar/Wifi.png" alt="Wifi symbol" />
+              </li>
+              <li>
+                <img
+                  class="images"
+                  src="bar/Battery.png"
+                  alt="battery symbol"
+                />
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="lower-bar">
+          <div class="haftamu">Haftamu</div>
+          <p class="allign">
+            <img src="bar/Enabled.png" alt="hamburger menu symbol" />
```

Comment:
- [ ] [Optional] On desktop size the hamburger menu button should not be visible, kindly remove the menu on the desktop view for best User experience

Reply:
#ismailco this is only the mobile version and the assignment is on mobile version.